packages/web-components-config-server/src/routes/web-components-config/__mocks__/web-components-config.ts

```diff
@@ -1,24 +1,12 @@
 export const webComponentConfig = {
-  configuration: {
-    timeIntervalMinutes: 10,
-    days: ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'],
```

Comment:
Sorry to be annoying but can we leave in days? Just realised we need this, sorry!

Reply:
Maybe save as numbers so we can use with dayjs easily?

Reply:
Sorry, do you mean to leave it in mock or what?

Reply:
Sorry not clear from the file - I meant to leave days as a schema item. I had missed it in when I wrote the ticket - as I say, my bad